mkapp/app/setting.ini

```diff
@@ -41,3 +41,109 @@ enable=0
 
 [osd]
 embedded_mode=0
+
+#### OSD elements ####
+# top fan speed
+element_show_topfan_speed=1
```

Comment:
nit-pick: could we go `element_<name>_show` etc? would make this a little more readable imho